app/mutations/attachment/archiveProjectAttachmentFormChange.ts

```diff
@@ -0,0 +1,24 @@
+import { archiveProjectAttachmentFormChangeMutation } from "__generated__/archiveProjectAttachmentFormChangeMutation.graphql";
+import useMutationWithErrorMessage from "mutations/useMutationWithErrorMessage";
+import { graphql } from "react-relay";
+
+const mutation = graphql`
+  mutation archiveProjectAttachmentFormChangeMutation(
+    $input: UpdateFormChangeInput!
+    $connections: [ID!]!
+  ) {
+    updateFormChange(input: $input) {
```

Comment:
I tried to make the name the same as what it does but since we already use `discard` for archiving some other form changes, I'm going to change it to `discardProjectAttachmentFormChange`.
Also, I'm assuming we want to archive form changes and `deleteFormChange` deletes the form change.

Reply:
Gotcha, I do see delete in other mutations but that must be for the undo/discard functionality

Reply:
I think we only use it in a few places where we don't care about the archiving changes (we don't want the record history) and we can safely remove the form change.